### PR TITLE
Update license to allow for extended usage

### PR DESCRIPTION
**Pull request by jamiehurewitz**
Removed specific references to E20 so the tool can be used for limited purposes in other versions of the software.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -5,17 +5,17 @@ With regard to the Mattermost Software:
...
 to modify this Software and publish patches to the Software. You agree that
 Mattermost and/or its licensors (as applicable) retain all right, title and
 interest in and to all such modifications and/or patches, and all such
 modifications and/or patches may only be used, copied, modified, displayed,
-distributed, or otherwise exploited with a valid Mattermost Enterprise E20
-Edition subscription for the  correct number of user seats.  Notwithstanding
+distributed, or otherwise exploited with a valid license or Subscription for the correct number of 
+Registered Authorized Users of the Software.  Notwithstanding
 the foregoing, you may copy and modify the Software for development and testing
 purposes, without requiring a subscription.  You agree that Mattermost and/or
```
Comment (lieut-data) [inline LICENSE:20]:
```suggestion
the foregoing, you may copy and modify the Software for development and testing
purposes, without requiring a valid license or Subscription.  You agree that Mattermost and/or
```

**Review by lieut-data — CHANGES_REQUESTED** [review]
This is terrific, @jamiehurewitz! One thought below.

**Review by itao — APPROVED** [review]

**Comment by lieut-data** [comment]
@jamiehurewitz, from your perspective, is there anything else pending to move this from draft status and merging it?

**Comment by jamiehurewitz** [comment]
This should be ready to go @lieut-data

**Pull request merged**